prometheus-exporters/network-generic-ssh-exporter/templates/_config.yaml.tpl

```diff
@@ -17,6 +17,29 @@ lookup_sources:
       router_project: SELECT DISTINCT id, project_id FROM neutron.routers
 
 metrics:
+  zmq_rx_error:
+    regex: >-
+      ^ZmQ\:\s*RX\sCNT\:\s\d+\,\sBYTES\:\s\d+\,\sERRORS\:\s(\d*)
+    multi_value: true
+    value: $1
+    labels:
+      errors: $1  
+    description: check for zmq rx errors
+    metric_type_name: gauge
```

Comment:
It looks like a counter?

Reply:
yes, counter would be better

Reply:
Suggested changes have been made to _config.yaml.tpl